src/PriceCalculatorKata/Utils/DiscountCalculations.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+
+namespace PriceCalculatorKata
+{
+    class DiscountCalculations : Calculations
+    {
+        private double Cap = double.MaxValue;
```

Comment:
this is more of a `Constant`. The value never changes and can be computed only once.

Reply:
Do you mean treating cap as a constant?
I think it's changes when calling AssignCapAmount method

Reply:
ah ok nvm